index.js

```diff
@@ -3,10 +3,26 @@ const { readFile } = require('node:fs/promises')
 const fp = require('fastify-plugin')
 const { accessSync, existsSync, mkdirSync, readdirSync } = require('node:fs')
 const { basename, dirname, extname, join, resolve } = require('node:path')
-const HLRU = require('hashlru')
+const { Lru } = require('toad-cache')
```

Comment:
Great, please try importing and using _LruMap_, which should perform better

Reply:
I benchmarked with the current benchmarks and no difference in the benchmark cases, sgtm to if prefer it!